_drgn.pyi

```diff
@@ -1953,6 +1959,51 @@ class TypeKind(enum.Enum):
     FUNCTION = ...
     """Function type."""
 
+class SymbolBinding(enum.Enum):
+    "A ``SymbolBinding`` represents the linker visibility of a symbol."
+
+    UNKNOWN = ...
+    """The binding type is unknown."""
+
+    LOCAL = ...
+    """The symbol is local to its module."""
+
+    GLOBAL = ...
+    """The symbol is global."""
+
+    WEAK = ...
+    """The symbol is weak and may be overridden."""
+
+    UNIQUE = ...
+    """The symbol is unique among all global and local symbols."""
+
+class SymbolKind(enum.Enum):
+    "A ``SymbolKind`` represents what sort of value a symbol represents."
+
+    NOTYPE = ...
```

Comment:
I think that makes sense, since I did overload the NOTYPE value with the meaning of UNKNOWN. I'll admit I don't have any concept of what NOTYPE gets used for, so I'd also be willing to create a separate UNKNOWN so we can differentiate. But I'll stick with renaming it for now.